cosmo_tester/framework/testenv.py

```diff
@@ -265,6 +265,13 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def wait_for_expected_outputs(self, expected_outputs, timeout=3):
+        def assertion():
+            outputs = self.client.deployments.outputs.get(self.test_id)
+            outputs = outputs['outputs']
+            self.assertEqual(str(expected_outputs), str(outputs))
```

Comment:
why the use of str()? if these objects are dictionary there are different methods for comparing those